extensions/functions_string.yaml

```diff
@@ -248,3 +248,192 @@ scalar_functions:
             name: "replacement"
             description: The replacement string.
         return: "varchar<L1>"
+  -
+    name: concat_ws
+    description: Concatenate strings together separated by a separator.
+    impls:
+      - args:
+          - value: "string"
+            name: "string_arguments"
+            description: Strings to be concatenated.
+            variadic:
```

Comment:
I'm unsure if this is the proper format for variadic.  It should only apply to the "string_arguments"

Reply:
The variadic key is pary of the implemenation structure, not the argument structure. It always applies to the last argument, so only it can be made variadic.

Reply:
updated thanks!